components/brave_vpn/browser/BUILD.gn

```diff
@@ -47,6 +47,7 @@ source_set("unit_tests") {
 
   deps = [
     ":browser",
+    "connection:unit_tests",
```

Comment:
I mean this target only need to be included in desktop because connection module is only valid on desktop?

Reply:
done